pkg/bindings/manifests/manifests.go

```diff
@@ -117,6 +117,26 @@ func Remove(ctx context.Context, name, digest string, _ *RemoveOptions) (string,
 	return Modify(ctx, name, []string{digest}, optionsv4)
 }
 
+// Delete removes specified manifest from local storage.
+func Delete(ctx context.Context, name string) (*entities.ManifestRemoveReport, []error) {
```

Comment:
Why does it return an array of errors? Will there ever be a case where it returns more than one error?

Reply:
@Luap99 Since API returns a list of errors so just making sure bindings honor it we can match the struct to upstream API if we decide to change the API https://docs.podman.io/en/latest/_static/api.html?version=v4.0#operation/ManifestDeleteLibpod

Reply:
You need to wait til 5.0 to break existing API.

Reply:
can we use `pkg/errorhandling/JoinErrors` to get a single error?

Reply:
@giuseppe SGTM we can do this at binding end.

Reply:
> can we use `pkg/errorhandling/JoinErrors` to get a single error?

Amended with this suggested change.